app/models/manageiq/providers/amazon/inventory/parser/cloud_manager.rb

```diff
@@ -273,11 +273,11 @@ def instance_hardware(persister_instance, instance, flavor)
       :bitness              => architecture_to_bitness(instance['architecture']),
       :virtualization_type  => instance['virtualization_type'],
       :root_device_type     => instance['root_device_type'],
-      :cpu_sockets          => flavor[:cpus],
+      :cpu_sockets          => flavor[:vcpu],
```

Comment:
Before these were using the result of `persister.flavors.find(instance_type)` which was returning the "parsed" flavor.

The flavors returned by the collector (from db/fixtures/aws_instance_types.yml) have slightly different keys